rest_api/v2/components/index.yml

```
@@ -465,6 +499,43 @@ schemas:
     type: array
     items:
       $ref: '#/schemas/Contact'
+  GroupId:
+    description: Group id
+    type: number
+    example: 42
+  GroupName:
+    description: Group Name
+    type: string
+    example: ComfyZone Paris
+  GroupParentId:
+    description: Group Parent Id
```

```suggestion
    description: Group Parent Id. If not null, it means it's a subgroup.
```
A link to the subgroup description would be great